src/utils/connectedApps.ts

```diff
@@ -9,24 +9,32 @@ export const isValidUrl = (url: string) => {
 
 export const getDappHost = (url: string) => {
   if (!url) return '';
-  const host = new URL(url).host;
-  if (host.indexOf('www.') === 0) {
-    return host.replace('www.', '');
+  try {
+    const host = new URL(url).host;
+    if (host.indexOf('www.') === 0) {
+      return host.replace('www.', '');
+    }
+    return host;
```

Comment:
small nitpick, but the check is redundant and unnecessary. Instead just call `.replace` on it no matter what
```suggestion
    const host = new URL(url).host;
    return host.replace('www.', '');
```

Reply:
I think technically www can be a subdomain that isn't necessarily at the start of the URL